modules/ROOT/pages/dataweave-cookbook-change-script-output-mime.adoc

```diff
@@ -0,0 +1,113 @@
+= Change a Script's MIME Type Output
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, dataweave, data weave, output structure, mime type, mime
+
+This DataWeave example shows how to differentiate the output MIME type from the MIME type in which the output data is formatted. Such differentiation enables a Mule application to output a MIME type that clients can use to determine how to render the application's output data. An example is HTTP, which uses the MIME type to generate its `Content-Type` header.
+
+//LINK TO DW 1.0 EXAMPLES:
+include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
+
+The following example uses the "problem details" standard for APIs. The standard requires the
```

Comment:
"The following example" intro is a bit far from the actual presentation of the example. I wonder if any of the text in between could be better presented following the presentation? I am sort of partial to the following approach to examples:

The following example shows this main thing:
EXAMPLE
As shown in this example, blah blah blah and bladity blah blah.

Reply:
replaced with "The first example"